### PR TITLE
David

### PR DESCRIPTION
**Pull request by avidmaverick**
I have done the assignments for Day 3!

### CODE REVIEW

**Comment by avidmaverick** [comment]
Corrections Done!

**Comment by avidmaverick** [comment]
sorry. premature celebration :(

File: src/david/feature.js [hunk 1]
```diff
@@ -1 +1,92 @@
...
+(5 < 17) -- boolean
+null undefined
+3.142 -- number
+"Pi"-- string
+67+18 = number
+*/
+
+//convert number to base 2 
+var base2converter = function(number) {
+    var i= parseInt(number, 10).toString(2);
```
Comment (andela-ioraelosi) [inline src/david/feature.js:17]: Do remember to surround all your operators with spaces.

File: src/david/feature.js [hunk 2]
```diff
@@ -1 +1,92 @@
...
+    console.log(i);
```
Comment (andela-ioraelosi) [inline src/david/feature.js:18]: It's recommended that you redefine your functions to use the `return` keyword instead of the `console.log` method.

**Comment by andela-ioraelosi** [comment]
Thanks for making corrections to your code. There are a few other issues, however. I've left comments to guide you.

Remember to use semi-colons at the end of expressions.

**Pull request closed without merging**